static/js/message_events.js

```diff
@@ -259,6 +260,15 @@ export function update_messages(events) {
                 compose_fade.set_focused_recipient("stream");
             }
 
+            const current_drafts = drafts.draft_model.get();
+            for (const draft_id of Object.keys(current_drafts)) {
+                const draft = current_drafts[draft_id];
+                if (draft.topic === orig_topic) {
```

Comment:
This seems to not check that the stream matches; is that intentional? See also `util.same_topic`, which does a case-insensitive comparison.

Reply:
Ah yes thanks, I'm still wrapping my brain around the "topics as strings that must exist alongside a stream name" concept, and its implications. I'll update this.

Reply:
I'm assuming you were referring to `util.same_stream_and_topic`

Reply:
Yep!